### DISCUSSION TITLE
Data Locality / Replication

### DISCUSSION THREAD

**Discussion by BnJam** [microsoft/PlanetaryComputer#210] [General]
EO: Earth Observation
PC: Planetary Computer

I understand the following question would have a fairly unrealistic solution, however, I'd like to open the discussion on the topic regardless. 

Due to Microsoft Azure's "high consumption of SPOT instances" and being unable to get our own compute (PC Hub is excellent but my team's platform has specific tooling for our work), are there any plans in the roadmap to decentralize/replicate the PC datasets across to other regions/continents from West Europe? 

I ask as network costs along with full-priced nodes are financially unappealing for small- to mid-sized companies and having data available in multi-region setup could better distribute to regions where compute capacity is available while eliminating/reducing network costs of processing EO data.

**Comment by TomAugspurger** [comment]
We've discussed this, but don't have any immediate plans to.

Note that some of our datasets are available in other regions, and this is indicated in the `msft:region` property on the STAC collection. For example, https://planetarycomputer.microsoft.com/api/stac/v1/collections/noaa-climate-normals-tabular is in the `eastus` region.

**Comment by BnJam** [comment]
I see some value in adding to the search functionality of the web `dataset filter` for regions and/or other STAC Catalog/Collection properties. 

What do you think?

**Comment by mmcfarland** [comment]
Hi @BnJam, I do think there is some value in extending the dataset filter to search against a more comprehensive list of attributes. However, it's not a near-term roadmap priority. I'll note that the attribute is displayed on a dataset's detail page, so should be quickly visible while browsing the catalog.

![image](https://user-images.githubusercontent.com/1014341/233367762-780ff142-4036-4f41-8320-2554f8e6ed66.png)
